elevate-cpanel

```diff
@@ -871,14 +904,26 @@ sub run_stage_3 ($self) {
 
             $self->run_once(
                 install_pkgs => sub {
-                    unless ( Cpanel::Pkgr::is_installed('elevate-release') ) {
+                    my $leap_data_pkg = 'leapp-data-almalinux';
+                    if ($self->upgrade_to_rocky()) {
+                        $leap_data_pkg = 'leapp-data-rocky';
+                    }
+                    elsif ($self->upgrade_to_cloudlinux()) {
+                        $leap_data_pkg = 'leapp-data-cloudlinux';
+                    }
 
+                    if ($self->upgrade_to_cloudlinux()) {
```

Comment:
`elsif`?

Reply:
Condensed the conditionals. Does the new version look fine?